app/src/main/java/com/courseproject/tindar/usecases/conversation_list/ConversationResponseModel.java

```diff
@@ -1,4 +1,4 @@
-package com.courseproject.tindar.usecases.conversations;
+package com.courseproject.tindar.usecases.conversation_list;
```

Comment:
our package naming convention is all lowercases without underscore. Can you please use conversationlist for the package name?

Reply:
Done.